exercises/practice/resistor-color-duo/src/ResistorColorDuo.elm

```diff
@@ -0,0 +1,6 @@
+module ResistorColorDuo exposing (Color(..), value)
```

Comment:
Same as in the other PR, I think we should provide the type definition in the stub. It's just busy work to add them anyway.